subsys/bluetooth/mesh/app_keys.h

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_SUBSYS_BLUETOOTH_MESH_APP_KEYS_H_
+#define ZEPHYR_SUBSYS_BLUETOOTH_MESH_APP_KEYS_H_
+
+#include <bluetooth/mesh.h>
+#include "net_keys.h"
+
+/** Mesh Application. */
+struct bt_mesh_app {
```

Comment:
I understand the desire to keep this short, but this really isn't a "Mesh Application" - it's keys to be used for application layer of the stack. So I'd append _key to the name.

Reply:
I had to go through the spec again, looks like you're right. My motivation for this was mainly to create a clear distinction between the keys used for decrypting messages in transport (appkeys and devkeys) and the application keys you bind your models to. The spec appears to not have a proper name for `appkey+devkey` though, so I suppose we shouldn't try to force it. Will revert to `app_key` throughout.

Reply:
Done